driver_test.go

```diff
@@ -338,3 +309,302 @@ func TestIsDdl(t *testing.T) {
 		}
 	}
 }
+
+func TestExecContextDml(t *testing.T) {
+
+	// Open db.
+	ctx := context.Background()
+	db, err := sql.Open("spanner", dsn)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer db.Close()
+
+	// Set up test table.
+	_, err = db.ExecContext(ctx,
+		`CREATE TABLE TestExecContextDml (
+			key	INT64,
+			testString	STRING(1024),
+			testBytes	BYTES(1024),
+			testInt	INT64,
+			testFloat	FLOAT64,
+			testBool	BOOL
+		) PRIMARY KEY (key)`)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	type testExecContextDmlRow struct {
+		key        int
+		testString string
+		testBytes  []byte
+		testInt    int
+		testFloat  float64
+		testBool   bool
+	}
+
+	tests := []struct {
+		name      string
+		input     string
+		query     string
+		setUp     string
```

Comment:
I think these query names could be improved. It seems odd to have `input` and `query`. Please update these to clarify that they are query and what they are for. Currently `query` does not indicate what it is used for.

Reply:
Maybe a good representation would be a `Given When Then` (https://martinfowler.com/bliki/GivenWhenThen.html)

Reply:
I tried to get the energy of that while having them semi match the other tests with givenInput whenQueried wantRows (is 10 second search and replace if thats confusing though🙂)